.github/workflows/nightlies.yml

```diff
@@ -63,3 +63,22 @@ jobs:
             --user scipy-wheels-nightly \
             --skip-existing \
             dist/matplotlib-*.whl
+
+      - name: Remove old uploads to save space
+        shell: bash
+        run: |
+          N_LATEST_UPLOADS=5
```

Comment:
This comes from @ogrisel's suggestion https://github.com/matplotlib/matplotlib/issues/22757#issuecomment-1142412584 to

> only keep the 5 most recent dev wheels for a given project and platform spec

but the number here is arbitrary.